package.json

```diff
@@ -41,7 +41,7 @@
     "jsonwebtoken": "^3.2.2",
     "mime": "^1.2.11",
     "moment": "^2.9.0",
-    "mongoose": "^3.8.16",
+    "mongoose": "3.8.16",
```

Comment:
This one works and since the next step will be moving to node 6 and update all the dependencies